File: remove-bg.js
async function removeBackground() {
  const Jimp = (await import('jimp')).default;

  try {
    // Read the image
    const image = await Jimp.read('./public/fulfilment-logo.png');

    // Get the background color from the top-left corner
    const bgColor = image.getPixelColor(0, 0);

    // Scan through all pixels
    image.scan(0, 0, image.bitmap.width, image.bitmap.height, function(x, y, idx) {
      // Get the current pixel color
      const pixelColor = this.getPixelColor(x, y);

      // Calculate color distance (simple RGB distance)
      const rDiff = Math.abs(Jimp.intToRGBA(pixelColor).r - Jimp.intToRGBA(bgColor).r);
      const gDiff = Math.abs(Jimp.intToRGBA(pixelColor).g - Jimp.intToRGBA(bgColor).g);
      const bDiff = Math.abs(Jimp.intToRGBA(pixelColor).b - Jimp.intToRGBA(bgColor).b);

      // If the color is close to the background color, make it transparent
      if (rDiff < 20 && gDiff < 20 && bDiff < 20) {
        this.bitmap.data[idx + 3] = 0; // Set alpha to 0 (transparent)
      }
    });

    // Save the modified image
    await image.writeAsync('./public/fulfilment-logo.png');
    console.log('Background removed successfully!');
  } catch (error) {
    console.error('Error processing image:', error);
  }
}

removeBackground();

File: remove_bg.py
from PIL import Image
import numpy as np

# Load the image
img = Image.open('public/fulfilment-logo.png')
img = img.convert("RGBA")

# Get image data
data = np.array(img)

# Get the background color (assuming it's the color at position 0,0)
bg_color = data[0, 0][:3]  # RGB values only

# Create a mask for pixels matching the background color
# Allow small tolerance for anti-aliasing
tolerance = 10
mask = np.all(np.abs(data[:, :, :3] - bg_color) <= tolerance, axis=2)

# Set alpha channel to 0 for background pixels
data[:, :, 3] = np.where(mask, 0, data[:, :, 3])

# Create new image from modified data
new_img = Image.fromarray(data, 'RGBA')

# Save the new image
new_img.save('public/fulfilment-logo.png')
print("Background removed successfully!")